lib/matplotlib/patches.py

```diff
@@ -1654,6 +1654,30 @@ def get_corners(self):
         return self.get_patch_transform().transform(
             [(-1, -1), (1, -1), (1, 1), (-1, 1)])
 
+    def get_vertices(self):
+        """
+        Return the left and right vertex coordinates of the ellipse.
+
+        The definition can be found `here<https://en.wikipedia.org/wiki/Ellipse>`_
```

Comment:
```suggestion
        The definition can be found `here <https://en.wikipedia.org/wiki/Ellipse>`_
```
reStructuredText is picky ... This solves your doc build failure.